### PR TITLE
Use str representation of sorted tuples to be compatible with literal eval

### PR DESCRIPTION
**Pull request by tomvanmele**
### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas.datastructures.Mesh`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/compas/datastructures/volmesh/core/halfface.py [hunk 1]
```diff
@@ -934,7 +934,7 @@ def unset_edge_attribute(self, edge, name):
         u, v = edge
         if u not in self._plane or v not in self._plane[u]:
             raise KeyError(edge)
-        key = "-".join(map(str, sorted(edge)))
+        key = ",".join(map(str, sorted(edge)))
```
Comment (Licini) [inline src/compas/datastructures/volmesh/core/halfface.py]: Sorry if this has already been discussed, why does halfface edge attribute key not using tuples as well?

**Review by Licini — COMMENTED** [review]
lgtm

File: src/compas/datastructures/volmesh/core/halfface.py (hunk 1 above)
Comment (brgcode) [reply src/compas/datastructures/volmesh/core/halfface.py:937]: sorry seem to not have pushed latest vers yet :)

Comment (tomvanmele) [reply src/compas/datastructures/volmesh/core/halfface.py]: will change this as well

**Review by gonzalocasas — APPROVED** [review]
lgtm but I would add a couple of simple unit tests

**Pull request merged**